src/test/java/teammates/test/cases/ui/InstructorFeedbackEditSaveActionTest.java

```diff
@@ -80,8 +81,9 @@ public void testExecuteAndPostProcess() throws Exception{
         ar = (AjaxResult) a.executeAndPostProcess();
         pageData = (InstructorFeedbackEditPageData) ar.data;
         
-        expectedString = "The start time for this feedback session cannot be "
-                         + "earlier than the time when the session will be visible.";
+        expectedString = String.format(FieldValidator.TIME_FRAME_ERROR_MESSAGE,
+                                       FieldValidator.START_TIME_FIELD_NAME,
+                                       FieldValidator.SESSION_VISIBLE_TIME_FIELD_NAME);
```

Comment:
This is one error message where `String.format()` is truly needed because the error message reflects the conditional check of proper date precedence between four variables: submission opening, submission closing, submission visibility start, and responses visibility start.